src/main/java/au/org/ala/biocache/util/QueryFormatUtils.java

```diff
@@ -208,26 +208,22 @@ public void setMaxBooleanClauses(int maxBooleanClauses) {
      * fq = (key:val1 OR key:val2) or fq = (-key:val1 OR -key:val2)
      * fq = -(key:val1 OR key:val2) or fq = -(-key:val1 OR -key:val2)
      *
+     * a new fq format added 18/09/2020
+     * fq = license:"CC BY NC" license:"CC BY NC 4.0 (Int)"
+     *
      * inclusive and exclusive keys can't co-exist in one fq
      */
     private String parseFQ(String fq) {
         fq = StringUtils.trimToEmpty(fq);
-        if (StringUtils.isNotEmpty(fq)) {
-            boolean globalInclusive = true;
-
-            // if () or -()
-            int start = fq.indexOf('(');
-            int end = fq.indexOf(')');
-            if ((start != -1) ^ (end != -1)) return null;
+        if ((fq.startsWith("-(") || fq.startsWith("(")) && !fq.endsWith(")")) return null;
 
-            if (start != -1) {
-                if (fq.charAt(0) == '-') {
-                    globalInclusive = false;
-                }
-                fq = fq.substring(start + 1, end);
-            }
+        boolean globalInclusive = true;
+        if (fq.startsWith("-(") && fq.endsWith(")")) {
+            fq = fq.substring(2, fq.length() - 1);
+            globalInclusive = false;
+        } else if (fq.startsWith("(") && fq.endsWith(")")) fq = fq.substring(1, fq.length() - 1);
```

Comment:
again, as per the [ALA coding](https://docs.google.com/document/d/1AMGskNqG5yxShiGsfnUj7N1zYvQCDyp5TTOyFakNO8Y/edit#heading=h.ko3sk8algg53) standards please don't user single line control statement and use braces around control statements.